monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -73,6 +72,7 @@ class ReportService:
     _credentials_repository: Optional[ICredentialsRepository] = None
     _machine_repository: Optional[IMachineRepository] = None
     _node_repository: Optional[INodeRepository] = None
+    _report: Optional[Dict[str, Dict]] = None
```

Comment:
Set the default to `{}` since it's what the resources expect as an empty report?